src/components/navigation/NavBar.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../constants.scss";
+
+//DESKTOP
```

Comment:
I think we can remove these comments as the media query should be explanatory enough :)

Reply:
Removed